src/analyze/gui/mainwindow.cpp

```diff
@@ -404,6 +406,26 @@ MainWindow::MainWindow(QWidget* parent)
     m_ui->menu_File->addAction(m_closeAction);
     m_quitAction = KStandardAction::quit(qApp, SLOT(quit()), this);
     m_ui->menu_File->addAction(m_quitAction);
+    QShortcut* shortcut = new QShortcut(QKeySequence(QKeySequence::Copy), m_ui->stacksTree);
+    connect(shortcut, &QShortcut::activated,
```

Comment:
always use the four-param connect syntax, i.e. add another context arg (`this` should do)

Reply:
Just tried: 4-param connect syntax doesn't work with lamdba.

Reply:
Why?

http://doc.qt.io/qt-5/qobject.html#connect-5

Reply:
ok. my bad!